packages/cli/src/util.ts

```diff
@@ -32,8 +32,8 @@ import * as list from "./list";
  * messages.
  */
 export function validateClusterNameOrDie(cn: string): void {
-  if (cn.length > 13) {
-    die(`cluster name must not be longer than 13 characters`);
+  if (cn.length > 21) {
+    die(`cluster name must not be longer than 21 characters`);
```

Comment:
@spahl @sreis @jgehrcke any issues with this?

Reply:
It was originally set to 13 Chars because that was the limit for the BigTable instance name (when we had it for Loki/Cortex). Since that's no longer used then this should be fine. We may hit some issue elsewhere with another cloud service that has aggressive char limits on their Ids, since we use the cluster name in _most_ cloud infra Ids.

I'd suggest testing this with a cluster name that uses 21 chars in AWS and GCP

Reply:
Let's please update https://github.com/opstrace/opstrace/issues/24